### DISCUSSION TITLE
How do I reference a 'version' of this for inclusion in a corporate package system (ProGet)?

### DISCUSSION THREAD

**Discussion by benfrain** [csstools/postcss-plugins#224] [Q&A] [answered]
How do I get something like a tgz of the postcss-preset-env project to include into something like ProGet (basically a local npm)?

If I want to get something like the 7.3.1 version for example. Typically I would use a tag or release.

Apologies if I am coming at this backwards - first time attempting to use it!

**Comment by romainmenke** [comment, marked as answer]
Hi there!

I think you can use `npm info --json postcss-preset-env@7.3.1` to get a tarball url.

`npm info --json postcss-preset-env@7.3.1 | jq .dist.tarball`

-> `https://registry.npmjs.org/postcss-preset-env/-/postcss-preset-env-7.3.1.tgz`

I have no idea how ProGet works so this might not be helpful :)
This archive will not contain any dependencies.
So you will probably need to do the same for each.

**Reply by benfrain** [reply]
That's very helpful thanks. It was more just getting a useable url so that should be fine. As you can imagine, getting all the dependencies is always a fun task but hey, ho!
